services/frontend/src/models/BlockProducer/mutation_update_user_rating.js

```diff
@@ -3,15 +3,15 @@ import gql from 'graphql-tag'
 const MUTATION_UPDATE_USER_RATING = gql`
```

Comment:
@tetogomez 

With the hapi endpoint we will no longer need this mutation. We will call a trigger over http on the server that will take the `user` and `bp`  then sync the postgres DB with the blockchain tables.

for example: `https://hapi.eosrate.io/ratebp?producer=junglemorpho&user=eosratetest1`

Reply:
Thanks @xavier506, I prefer to merge this PR first and then create a new task to do that switch (graphql mutation to hapi)